lib/config/fileconf.go

```diff
@@ -246,17 +251,21 @@ func ReadConfig(reader io.Reader) (*FileConfig, error) {
 func MakeSampleFileConfig() (fc *FileConfig) {
 	conf := service.MakeDefaultConfig()
 
+	// generate a secure random token
+	randomJoinToken, err := utils.CryptoRandomHex(randomTokenLenBytes)
+	if err != nil {
+		panic(err)
```

Comment:
We shouldn't panic in our code, update the function to return an error and return the error here.